### PR TITLE
Typos fix and minor texts clarifying

### PR DESCRIPTION
**Pull request by khanhtc1202**
**What this PR does / why we need it**:
- fix typos
- clarify some texts
- replace `</br>` with `<br>`, suspend templates warnings on my IDE 😂 

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/news/announcing-pipecd/index.md [hunk 1]
```diff
@@ -102,4 +102,4 @@ PipeCD team is hiring engineers/interns to work on PipeCD. Please contact us if
 ### Thanks
 
 Finally, we would like to thank the existing open-source CD projects like Spinnaker, FluxCD, ArgoCD... PipeCD has been built on many great ideas from those great projects.
-PipeCD team would also like to thank CyberAgent's engineers and collaborators from other companies, who have sent us so many valuable feedback throughout the development process to this day.
+PipeCD team would also like to thank CyberAgent's engineers and collaborators from other companies, who have sent us so many valuable feedbacks throughout the development process to this day.
```
Comment (nghialv) [inline docs/content/en/blog/news/announcing-pipecd/index.md:105]: "feedback" is uncountable.

File: docs/content/en/docs/contribution-guidelines/architectural-overview.md [hunk 2]
```diff
@@ -13,13 +13,13 @@ Component Architecture
 
 ### Piped
 
-A single binary component that you run in your cluster, your local network to handle the deployment tasks.
+A single binary component run in your cluster, your local network to handle the deployment tasks.
```
Comment (nghialv) [inline docs/content/en/docs/contribution-guidelines/architectural-overview.md:16]: "runs"

**Comment by nghialv** [comment]
@khanhtc1202 Nice catch! Thank you.
 I have just left some comments.

File: docs/content/en/blog/news/announcing-pipecd/index.md (hunk 1 above)
Comment (khanhtc1202) [reply docs/content/en/blog/news/announcing-pipecd/index.md]: thanks, addressed on https://github.com/pipe-cd/pipe/pull/1111/commits/8ff128265b94202cfac3c8b48ca724da50739740.

File: docs/content/en/docs/contribution-guidelines/architectural-overview.md (hunk 2 above)
Comment (khanhtc1202) [reply docs/content/en/docs/contribution-guidelines/architectural-overview.md]: thanks, addressed on https://github.com/pipe-cd/pipe/pull/1111/commits/8ff128265b94202cfac3c8b48ca724da50739740.

**Comment by nghialv** [comment]
Nice.
/approve

**Pull request merged**